initial/contract-rest-client/src/test/java/hello/ContractRestClientApplicationTest.java

```diff
@@ -1,37 +1,36 @@
 package hello;
 
 import org.assertj.core.api.BDDAssertions;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.cloud.contract.stubrunner.junit.StubRunnerRule;
-import org.springframework.cloud.contract.stubrunner.spring.AutoConfigureStubRunner;
+import org.springframework.cloud.contract.stubrunner.junit.StubRunnerExtension;
 import org.springframework.cloud.contract.stubrunner.spring.StubRunnerProperties;
 import org.springframework.http.ResponseEntity;
-import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.web.client.RestTemplate;
 
-@RunWith(SpringRunner.class)
 @SpringBootTest
-@AutoConfigureStubRunner(
-		ids = "com.example:contract-rest-service:0.0.1-SNAPSHOT:stubs:8100",
-		stubsMode = StubRunnerProperties.StubsMode.LOCAL
-)
 public class ContractRestClientApplicationTest {
 
+	@RegisterExtension
```

Comment:
Why can't we use the annotation here?

Reply:
I would prefer it myself… I was just doing 'monkey see, monkey do' what was in the 'completed' directory. ;-)

I am still learning about the Spring Cloud things and do not know if one or the other method is a Method preferred by Spring. Personally I like the annotation - I will change it in the 'initial' and 'complete' example!